src/app/core/store/collections/collections.action.ts

```diff
@@ -0,0 +1,240 @@
+/*
+ * Lumeer: Modern Data Definition and Processing Platform
+ *
+ * Copyright (C) since 2017 Answer Institute, s.r.o. and/or its affiliates.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {Action} from '@ngrx/store';
+import {QueryModel} from '../navigation/query.model';
+import {AttributeModel, CollectionModel} from './collection.model';
+import {PermissionModel, PermissionType} from '../permissions/permissions.model';
+
+export enum CollectionsActionType {
+
+  GET = '[Collections] Get',
+  GET_SUCCESS = '[Collections] Get :: Success',
+  GET_FAILURE = '[Collections] Get :: Failure',
+
+  CREATE = '[Collections] Create',
+  CREATE_SUCCESS = '[Collections] Create :: Success',
+  CREATE_FAILURE = '[Collections] Create :: Failure',
```

Comment:
What about concatenating these constants ? For example: CREATE_SUCCESS = `${CREATE} :: Success` ?

Reply:
Do you think it will make the code more readable?

Reply:
I'm not sure about that. Since it's line by line, it can stay as it is.